modules/002-deckhouse/templates/deployment.yaml

```diff
@@ -213,7 +213,6 @@ spec:
               value: /tmp/.bash_history
             - name: DEBUG_HTTP_SERVER_ADDR
               value: "127.0.0.1:9652"
-            {{- include "helm_lib_envs_for_proxy" . | nindent 12 }}
```

Comment:
not sure about this change as we had some issues related to inter-pod communication in a multi-master cluster with strict http_proxy requirements. Non-leader pods couldn't reach the leader, resulting in failed readiness probes.

Reply:
I tested in an environment where there is full access between servers, but access to external network and internet works only through proxy.

Reply:
in that case, pods couldn't reach even the nodes' ip addresses. the solution was to start using `.cluster.local` domain name, which was  added to the http_proxy list,  to request the leader pod's status.